### PR TITLE
Add certifi.contents()

### PR DESCRIPTION
**Pull request by jdufresne**
Added in commit:
https://github.com/certifi/python-certifi/commit/3e585603ff265552c2e137d064ea092d51332602

Available since version 2020.04.05.

### CODE REVIEW

File: stubs/certifi/METADATA.toml [hunk 1]
```diff
@@ -1,2 +1,2 @@
-version = "0.1"
+version = "2020.4.5"
```
Comment (jdufresne) [inline stubs/certifi/METADATA.toml:1]:
```
Traceback (most recent call last):
  File "/home/runner/work/typeshed/typeshed/./tests/check_consistent.py", line 197, in <module>
    check_metadata()
  File "/home/runner/work/typeshed/typeshed/./tests/check_consistent.py", line 172, in check_metadata
    assert version.count(".") == 1, msg
AssertionError: Unsupported Python version 2020.4.5
Error: Process completed with exit code 1.
```

This is the format used by the project to represent versions: https://pypi.org/project/certifi/#history

What is the best way to handle this in typeshed?

Comment (hauntsaninja) [reply stubs/certifi/METADATA.toml:1]: I think the way it works right now is you just use "2020.4" instead.

Comment (jdufresne) [reply stubs/certifi/METADATA.toml]: Sounds good. I adjusted that in the latest revision.

**Comment by hauntsaninja** [comment]
As always, thank you!

**Pull request merged**